tasks/src/video-classification/about.md

```diff
@@ -0,0 +1,70 @@
+## Use Cases
+Video classification models can be used to categorize what a video is all about.
+
+### Activity Recognition
+Video classification models are used to perform activity recognition which are useful for fitness applications. Activity 
+recognition is also helpful for vision-impaired individuals especially when they're commuting.
+
+### Surveillance
+
+Video classification models are employed to power many surveillance applications to keep track of different activities
```

Comment:
I think this is an ethical concerning use case and we should not have this as a highlighted use case